bundles/org.openhab.binding.caddx/src/main/resources/OH-INF/thing/zone.xml

```diff
@@ -11,6 +11,7 @@
 
 		<label>Caddx Alarm Zone</label>
 		<description>Represents a physical device such as a door, window, or motion sensor.</description>
+		<semantic-equipment-tag>AlarmDevice</semantic-equipment-tag>
```

Comment:
See discussion in other PR. 
1. Both are tagged different.
2. In both casses i opt to introduce a new tag: `AlarmZone`

Reply:
Ok. But maybe just `Zone` rather than `AlarmZone` ?? .. e.g. Philips Hue has lighting zones .. and I could imagine heating zones etc. => WDYT?

Reply:
That would be a discussion with core and add ons maintainers. For both options there are pros and cons. 

If Zone would be the highest level. We could always introduce descendants like alarm, heating or lightzone

Reply:
In https://github.com/openhab/openhab-core/pull/4738 I added both Zone and a child AlarmZone

Reply:
However this one is definitely an `AlarmDevice` so I am closing this comment